app/controllers/course/assessment/assessments_controller.rb

```diff
@@ -209,6 +221,19 @@ def ordered_assessments_by_tab
     tabs.sort_by { |tab_hash| tab_hash[:title] }
   end
 
+  def course_user_ids
+    case params[:course_users]
```

Comment:
I feel like this method isn't very defensive. Are we expecting `params[:course_users]` to be always specified? What if it contains some garbage value?

Reply:
Yes we are expecting it to be always specified. Hm.. In that case should we return an error if it's a garbage value?

Reply:
Would be good to return an error if it's not a value within the array of expected values.

Reply:
Yup updated!